packages/jest-message-util/src/index.ts

```diff
@@ -160,9 +164,12 @@ export const formatExecError = (
       ? '\n' + formatStackTrace(stack, config, options, testPath)
       : '';
 
-  if (blankStringRegexp.test(message) && blankStringRegexp.test(stack)) {
+  if (
+    (blankStringRegexp.test(message) && blankStringRegexp.test(stack)) ||
+    typeof stack !== 'string'
```

Comment:
do the `typeof` first here - that's way faster than `RegExp.test`

Reply:
done 👍🏻